lib/events/gcssessions/gcshandler.go

```diff
@@ -292,13 +303,27 @@ func (h *Handler) Download(ctx context.Context, sessionID session.ID, writerAt i
 	return nil
 }
 
-func (h *Handler) path(sessionID session.ID) string {
+// DownloadSummary downloads a session summary from a GCS bucket and writes the
+// result into a writer. Returns trace.NotFound error if the recording is not
+// found.
+func (l *Handler) DownloadSummary(ctx context.Context, sessionID session.ID, writer io.WriterAt) error {
+	return trace.NotImplemented("DownloadSummary is not implemented for GCS handler")
```

Comment:
Why did we implement this for everything but GCS?

Reply:
Mystery solved: the test is broken. :D Fixing it now.